src/components/util/AutocompleteInput/AutocompleteList.tsx

```diff
@@ -158,8 +158,11 @@ export const AutocompleteList = React.forwardRef<HTMLDivElement, AutocompleteLis
 
 		React.useEffect(() => {
 			const showAutocomplete = Boolean(
-				(alwaysShow || (anchor && value && value !== focusedOption)) && list.length,
+				(alwaysShow ||
+					(anchor && value && focusedOption?.toLocaleLowerCase().includes(value.toLowerCase()))) &&
```

Comment:
not sure why `toLocaleLowerCase` is used.

Autocomplete list accepts anchor, so what if we setAnchor to null upon selecting value, i think this might solve the problem